src/pack.iced

```diff
@@ -2,36 +2,69 @@ try
   mp = require 'msgpack'
 catch e
 
-if not mp?
-  try
-    pp = require 'purepack'
-    mp = pp
-  catch e
+try
```

Comment:
this is slightly different than before. it seems fine to me but just making sure this is ok

Reply:
purepack is still the default

Reply:
But the old code only checks if mp isn't there before. I don't care just pointing it out